lib/app.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:flutter/material.dart';
+import 'package:hiking4nerds/navigation/segment_navigator.dart';
+import 'navigation/bottom_navigation.dart';
+
+class App extends StatefulWidget {
```

Comment:
new file for our App. Much better than having everything inside the Main. Great!